src/modules/auth/index.ts

```diff
@@ -3,3 +3,4 @@ export { default as ForgetPasswordForm } from './ForgetPasswordForm/ForgetPasswo
 export { default as SignUpForm } from './SignUpForm/SignUpForm';
 export { default as ForgetPasswordSuccess } from './ForgetPasswordSuccess/ForgetPasswordSuccess';
 export { default as ResetPasswordForm } from './ResetPasswordForm/ResetPasswordForm';
+export { default as ErrorPage } from './ErrorPage/ErrorPage';
```

Comment:
Also don't forget to remove this line after you move the file to another folder

Reply:
deleted line 6